engine/src/test/java/com/datatorrent/stram/client/StramAppLauncherTest.java

```diff
@@ -98,7 +98,7 @@ public void testGetTokenRefreshKeytab() throws Exception
     {
       Configuration conf = new Configuration(false);
       File storeKeytab = new File(dfsDir, "keytab2");
-      conf.set(StramClientUtils.KEY_TAB_FILE, storeKeytab.getPath());
+      conf.set(StramClientUtils.TOKEN_REFRESH_KEYTAB, storeKeytab.getPath());
```

Comment:
@devtagare should not replace test for KEY_TAB_FILE as it is still supported even though it's usage is deprecated. It should be an additional test for TOKEN_REFRESH_KEYTAB.